elastic/kibana
# [Fleet] Adjust privileges for GET output and GET download_source endpoints

Fixes https://github.com/elastic/kibana/issues/191266

## Summary
Updating the authz for following endpoints:
- `GET /agent_download_sources`
- `GET /agent_download_sources/{id}`
- `GET /outputs`
- `GET /outputs/{id}`
They need to have `authz.fleet.readSettings || authz.fleet.readAgentPolicies` as they should be visible in the agent policy settings page as well.


### Testing
- Enable feature flag `subfeaturePrivileges`
- Create a role with following privileges:
![Screenshot 2024-10-04 at 15 49 54](https://github.com/user-attachments/assets/4bbc95e4-01d0-43e0-a539-b03b8f4c219e)
- Create a user that has the previous role
- Log in and go to any agent policy > settings
- The download source and output fields should be filled and editable. They were previously empty, as the GET endpoints were failing with 403 Forbidden

### Checklist
